### PR TITLE
Updated The READ.me File

### PR DESCRIPTION
**Pull request by ManojLL**
## Description of what I changed

Updated the READ.me file of the module
## Issue I worked on

see https://openmrs.atlassian.net/browse/FLAG-

## Checklist: I completed these to help reviewers :)

- [ ] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [ ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ ] My pull request is **based on the latest changes** of the master branch.
  
  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,109 @@
...
+`Flag Priority` - The flag priority determines the order of the display of the flags in the display
+points. and flags can be displayed in different colors using display styles. The display style
+should be a valid attribute that can be used with a HTML <span> tag.
+
+[Patient Flag Module wiki page](https://openmrs.atlassian.net/wiki/spaces/Archives/pages/25467777/Patient+Flags+Module#User-Guide)
+
+Developer Guide
+---------------
+
+#### Setup Project
```
Comment (wikumChamith) [inline README.md:40]: Lets use a ordered list for these steps

File: README.md [hunk 2]
```diff
@@ -1,4 +1,109 @@
...
+* Maven
+
+clone the project
+
+`git clone https://github.com/<username>/openmrs-module-patientflags.git
+`
+
+build the project 
+
+`mvn clean istall` in the root module package
```
Comment (wikumChamith) [inline README.md:54]: "install"

Comment (ManojLL) [reply README.md]: READ.me file updated

File: README.md [hunk 3]
```diff
@@ -1,4 +1,108 @@
...
+Developer Guide
+---------------
+
+**Prerequisite**
+
+* Java version ( 8, 11 or 17)
+* Maven
+#### Setup Project
+
+1. clone the project
```
Comment (wikumChamith) [inline README.md:46]: "Clone"

File: README.md [hunk 4]
```diff
@@ -1,4 +1,108 @@
...
+
+    `git clone https://github.com/<username>/openmrs-module-patientflags.git
+`
+
+2. build the project
```
Comment (wikumChamith) [inline README.md:51]: "Build"

File: README.md [hunk 5]
```diff
@@ -1,4 +1,108 @@
 openmrs-module-patientflags
-===========================
+==========================
 
-Provides a mechanism for marking patient records with important messages within OpenMRS
+The openMRS Patient Flag Module provides the functionality to generate warning flags /
```
Comment (wikumChamith) [inline README.md:4]: OpenMRS

File: README.md [hunk 6]
```diff
@@ -1,4 +1,108 @@
...
+
+**Prerequisite**
+
+* Java version ( 8, 11 or 17)
+* Maven
+#### Setup Project
+
+1. Clone the project
+
+    `git clone https://github.com/<username>/openmrs-module-patientflags.git
```
Comment (wikumChamith) [inline README.md:48]: Let's ask them to fork the repo first

**Pull request merged**